Lekce-8-HW/Program.cs

```diff
@@ -4,7 +4,70 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello, World!");
+
+            IDriveable[] driveables = new IDriveable[150];
+            for (int i = 0; i < 50; i++)
+            {
+                driveables[i] = new Car
+                {
+                    PocetDveri = 4,
+                };
+                ((Car)driveables[i]).SetBrand($"Car{i + 1}");
+                ((Car)driveables[i]).SetmileAge(1000 + i);
```

Comment:
Tady kombinujes dva pristupy pri vytvareni te instance, za me trochu nespravne.

Jsou dva zpusoby, jak bys tohle mel spravne udelat:
1. Vytvorit instanci `Car` a ty hodnoty predat do konstruktoru, ktery je pak priradi do properties. Nejak cca takto by to pak vypadalo: `new Car($"Car{i + 1}", 1000 + i, 4)`
2. zavolat prazdny konstruktor a hodnotu pro public property vyplnit explicitne (to je ten zapis, co pouzivas v kodu ty).

Zvolil jsi 2, coz je uplne ok, pak ale nedava smysl na dalsich radcich volat `SetBrand` a `SetmileAge`, kde musis komplikovane pretypovavat, znovu pristupovat k prvku v kolekci apod... jednodussi a spravnejsi by to bylo takhle (zvlast kdyz brand i mileage jsou public property):
```suggestion
                driveables[i] = new Car
                {
                    PocetDveri = 4,
                    Brand = $"Car{i + 1}",
                    Mileage = 1000 + i,
                };
```